src/CalendarViews/SharedStyling/WeekFormating.css

```diff
@@ -0,0 +1,5 @@
+.weekFormat {
+    display: grid;
+    grid-template-columns: calc(100% / 7) calc(100% / 7) calc(100% / 7) calc(100% / 7) calc(100% / 7) calc(100% / 7) calc(100% / 7);
```

Comment:
while this works, there are built in ways to repeatedly assign fractions of the width to a column.

```css
grid-template-columns: repeat(7, 1fr);
```
this is cleaner to read and if you add something like a gap later on it'll handle it no problem too.